frontend/src/features/permits/components/dashboard/PermitLists.tsx

```diff
@@ -14,7 +14,7 @@ export const PermitLists = React.memo(() => {
   const tabs = [
     {
       label: "Applications in Progress",
-      count: query?.applicationsInProgressQuery?.data?.items?.length,
+      count: query?.data?.items?.length,
```

Comment:
This is incorrect. Use `count: query?.data?.meta?.totalItems` instead

Reply:
updated - thanks